include/spdlog/tcp_sink.h

```diff
@@ -0,0 +1,47 @@
+
+#include <spdlog/spdlog.h>
+#include <spdlog/sinks/base_sink.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+using namespace std;
+template<typename Mutex>
+class tcp_sink : public spdlog::sinks::base_sink <Mutex>
+{
+public:
+    tcp_sink(std::string address,int port)
+    {
+        if ((sock = socket(AF_INET, SOCK_STREAM, 0)) < 0)
+        {
+            printf("\n Socket creation error \n");
```

Comment:
Please throw in case of errors
For example:

```c++
SPDLOG_THROW(spdlog_ex("Socket creation error", errno));
```

Reply:
Fixed